src/common/footer/Footer.module.css

```diff
@@ -0,0 +1,69 @@
+.koros {
+    width: 100%;
+    fill: var(--hds-ui-color-black-5);
+    height: 60px;
+}
+
+.footer {
+    width: 100%;
+    flex-shrink: 0;
+    background-color: var(--hds-ui-color-black-5);
+    padding: 0 10px 20px;
+}
+
+.content {
+    width: 1200px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+}
+
+@media (max-width: 1200px) {
```

Comment:
Could you make this style mobile-first, i.e. make `width: auto;` the default for `.content` and override that when the screen is wide enough `@media (min-width: 1200px) { .content { width: 1200px; } }`?

Reply:
Changed.